openapi/preprocess_spec.py

```diff
@@ -330,20 +330,27 @@ def main():
     )
     args = argparser.parse_args()
 
-    spec_url = 'https://raw.githubusercontent.com/%s/%s/' \
+
+    skip_fetch_spec = "OPENAPI_SKIP_FETCH_SPEC"
+    unprocessed_spec = args.output_spec_path + ".unprocessed"
+    in_spec = ""
+    if os.environ.get(skip_fetch_spec, False):
```

Comment:
Does this handle empty string correctly? The setting above will set the env variable to "", right?

It might be good to have tests for this code (though I realize there aren't a lot of tests right now...)

Reply:
the script will read from the local spec as long as the env var is set, including empty string

Reply:
Right, but that means that by default this code will run, correct? I don't think that's what we want...

Reply:
> if os.environ.get(skip_fetch_spec) or False:

fixed.. now the empty env var will be considered as False